src/components/limited-restoration-panel/LimitedRestorationPanel.vue

```diff
@@ -0,0 +1,104 @@
+<template id="limited-restoration-panel">
+  <v-radio-group class="mt-0 pt-0 ml-8" v-model="selectMonths" mandatory>
+    <v-radio id="twentyFour-radio" name="selectMonths" label="Two Years" :value=24 />
+    <v-radio id="eighteen-radio" name="selectMonths" label="18 months" :value=18 />
+    <v-radio id="twelve-radio" name="selectMonths" label="12 months" :value=12 />
+    <v-radio id="six-radio" name="selectMonths" label="6 months" :value=6 />
+    <v-row class="ml-0">
+      <v-radio id="custom-months" name="selectMonths" label="Month(s): " value="customMonths">
+      </v-radio>
+      <v-text-field class="shrink mx-4" type="number" min=1 max=24 :rules="monthRules" v-model="numberOfMonths"
+        :disabled="!customMonths" @change="onMonthsChanged" filled />
+    </v-row>
+  </v-radio-group>
+</template>
+<script lang="ts">
```

Comment:
Typically we have a blank line between the template and script (and script and style), and no blank line at the top of the script block.

Reply:
Noted. Going to keep this in mind.